examples/index.html

```diff
@@ -14,7 +14,8 @@
   </style>
   <meta charset="utf-8">
   <link href="../index.css" rel="stylesheet">
-  <script type="module" src="../dist/index.esm.js"></script>
+  <!-- Link to `../dist/index.esm.js` for testing the module locally -->
```

Comment:
```suggestion
  <!-- <script type="module" src="../dist/index.esm.js"></script> -->
```

As we do for other elements, so in development we can uncomment these with text editor shortcuts:

https://github.com/github/auto-complete-element/blob/1de3674e948529e2e7f4e12bb1e539a284552e29/examples/index.html#L49-L50